packages/blockchain/egld/src/lib/__tests__/egld.offchain.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { REPLACE_ME_WITH_TATUM_API_KEY, TEST_DATA } from '@tatumio/shared-testing-common'
+import { TatumEgldSDK } from '../egld.sdk'
+
+describe('Offchain - tx', () => {
+  const sdk = TatumEgldSDK({
+    apiKey: REPLACE_ME_WITH_TATUM_API_KEY,
+  })
+
+  const testData = TEST_DATA.EGLD
+  // skiped because account doesn't have balance
```

Comment:
I am not sure how to fund this account because the address that is connected to the account has balance but it didn't affect the balance of the account. I can investigate more - will put TODO for now